projects/1.20/assets/tetra/tetra/lang/zh_cn.json

```
@@ -0,0 +1,2465 @@
+  "tetra.holo.craft.holosphere_shortcut_missing": "需要全息球",
+  "tetra.holo.craft.holosphere_shortcut_unavailable": "无法获得属性",
+  "tetra.holo.craft.materials.stat.primary": "硬度",
+  "tetra.holo.craft.materials.stat.primary.short": "硬度",
+  "tetra.holo.craft.materials.stat.primary.tooltip": "材料的硬度、锋利度，被塑形成硬质或锋利边缘的能力",
+  "tetra.holo.craft.materials.stat.secondary": "密度",
+  "tetra.holo.craft.materials.stat.secondary.short": "密度",
+  "tetra.holo.craft.materials.stat.secondary.tooltip": "材料的密度、刚性或重量，被挥动时动能大小",
+  "tetra.holo.craft.materials.stat.tertiary": "韧性",
+  "tetra.holo.craft.materials.stat.tertiary.short": "韧性",
+  "tetra.holo.craft.materials.stat.tertiary.tooltip": "材料的韧性或抗拉强度，材料被弯曲后回弹的速度",
+  "tetra.holo.craft.materials.stat.tool_level": "工具等级",
+  "tetra.holo.craft.materials.stat.tool_level.short": "等级",
+  "tetra.holo.craft.materials.stat.tool_level.tooltip": "使用这种材料制作的组件的基础等级",
+  "tetra.holo.craft.materials.stat.tool_efficiency": "工具效率",
+  "tetra.holo.craft.materials.stat.tool_efficiency.short": "效率",
+  "tetra.holo.craft.materials.stat.tool_efficiency.tooltip": "使用这种材料制作的组件的基础工具效率",
+  "tetra.holo.craft.materials.stat.durability": "耐久度",
+  "tetra.holo.craft.materials.stat.durability.short": "耐久",
+  "tetra.holo.craft.materials.stat.durability.tooltip": "使用这种材料制作的组件的基础耐久度",
+  "tetra.holo.craft.materials.stat.integrity": "完整度",
+  "tetra.holo.craft.materials.stat.integrity.short": "完整度",
+  "tetra.holo.craft.materials.stat.integrity.tooltip": "使用这种材料制作的组件的基础完整度增益或消耗（增益丨消耗）",
+  "tetra.holo.craft.materials.stat.magic_capacity": "魔法容量",
+  "tetra.holo.craft.materials.stat.magic_capacity.short": "魔力",
+  "tetra.holo.craft.materials.stat.magic_capacity.tooltip": "使用这种材料制作的主要组件的魔法容量",
+  "tetra.holo.craft.materials.stat_modifier": "强化",
+  "tetra.holo.craft.materials.stat_modifier.tooltip": "使用这种材料制作的主要组件会提供§e%s§r强化：",
+  "tetra.holo.craft.materials.stat_effect": "效果",
+  "tetra.holo.craft.materials.stat_effect.tooltip": "使用这种材料制作的组件会提供§e%s§r效果：",
+  "tetra.holo.craft.materials.feature": "特性",
+  "tetra.holo.scan.status": "§8状态：§r%s",
+  "tetra.holo.scan.active": "激活",
+  "tetra.holo.scan.snoozed": "§8休眠 §r%s:%s",
+  "tetra.holo.quick_access.snooze_on": "§7使扫描器\n§r休眠",
+  "tetra.holo.quick_access.snooze_off": "§7使扫描器\n§r启动",
+  "tetra.holo.place": "§7[§8Shift+右键§7] §8放置",
+  "tetra.workbench.slot_detail.details_tab": "详情",
+  "tetra.workbench.slot_detail.craft_tab": "制作",
+  "tetra.workbench.slot_detail.tweak_tab": "调整",
+  "tetra.workbench.slot_detail.tweak_apply": "应用",
+  "tetra.workbench.module_detail.empty": "空槽位",
+  "tetra.workbench.module_detail.not_tweakable": "调整不可用",
+  "tetra.workbench.schematic_list.empty": "所选插槽没有可用的升级",
+  "tetra.workbench.schematic_detail.craft": "制作",
+  "tetra.workbench.schematic_detail.back": "返回",
+  "tetra.workbench.schematic_detail.experience": "此操作需要%d级经验",
+  "tetra.workbench.schematic_detail.destabilize_tooltip": "该物品有魔力去稳化的组件，进行此操作有可能对以下组件造成负面效果：",

```

| **审查级别**    |
|:-:|
| ❓疑问 / 需要作者确认 |

**原文: This item has magically unstable modules, performing this action has a chance to affect the following modules negatively:**

```suggestion
  "tetra.workbench.schematic_detail.destabilize_tooltip": "该物品有魔力去稳化的组件，进行此操作有可能对以下组件造成负面效果：",
```

---

Tetra的工具有魔力属性，可在Tetra工作台上工具面板详情查看
每个魔咒都需要消耗一定量的魔力，魔力消耗成负数也没问题，但代价是用多了会脱手，然后变成怪来打你
有些材料或改进（比如镀金、镶金黑石）能够提升魔力上限，而有些（诡异木、绯红木）会减少
此处出现位置是，往具有减少魔力上限组件或改进的工具上打超出剩余量的附魔书，会在确认按钮上显示

不过改成`魔力不稳定`怎么样

---

> 不过改成`魔力不稳定`怎么样

这个还不错

---

那先这个

---

后面还有两个键里用到了这个概念
`item.tetra.modular.magic_capacity.description_extended`（形容魔法改进的变化）
`tetra.improvement.destabilized/haunted.name`（这种改进的名称）
这两个键需要和这里统一，等看到后面再改改看，现在直接把“不稳定”放进去有些怪